newrelic-agent/src/main/java/com/newrelic/agent/config/SecurityAgentConfig.java

```diff
@@ -44,6 +48,8 @@ public class SecurityAgentConfig {
     private static final Config config = NewRelic.getAgent().getConfig();
     private static final String ENABLED = "enabled";
     private static final String DISABLED = "disabled";
+    public static final Set<String> SECURITY_AGENT_CLASS_TRANSFORMER_EXCLUDES_TO_IGNORE = Collections.unmodifiableSet(
+            Sets.newHashSet("^java/security/.*", "^javax/crypto/.*", "^net/sf/saxon.*"));
```

Comment:
This is the full set of default `class_transformer.exclude` rules that need to be ignored when the security agent is enabled. Should further exclude rules need to be treated the same in the future they can simply be added here and no other logic should need to be changed elsewhere in the agent codebase. Only new tests would need to be added to `IncludeExcludeTest`.